client/src/index.js

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 
 import ChatLink from './pages/chatlink';
-import Messaging from './pages/messaging';
+import Messaging from './pages/messaging/main';
```

Comment:
Any specific thought behind putting it inside `main` dir ?

Reply:
index.js and Style.module.css should have done ig. Was just separating them into a new folder as well